OlawaleJoseph/buddy-api
# Setup Test

## What does this PR do?
This PR sets up a test environment and integrates CI (Travis CI).

### Implementation
- Install and setup jest config file
- Integrate app with travis-ci.com and define test parameters in .travis.yml file
- Integrate Coveralls for test coverage
- Integrate Code Climate for test Maintainability

### How to test
- clone this repo `git clone git@github.com:OlawaleJoseph/buddy-api.git`.
- cd into the cloned repo then checkout into `ch-setup-test` branch.
- Open a terminal and run `npm run test`;
- Check the badge in the README.md file for the real-time status of the repo.
